docs/dg/dev/third-party-module-development/develop.md

```diff
@@ -0,0 +1,181 @@
+---
+title: How to develop a Third-Party Spryker module
+description: How to develop a Spryker module
+last_updated: Jun 7, 2024
+template: howto-guide-template
+---
+
+## Architectural conventions
+
+Follow the [architectural conventions](/docs/dg/dev/architecture/architectural-convention.html) tagged with *Module Development*.
+
+## Enable a custom namespace
+
+To enable a custom namespace, adjust **config/Shared/config_default.php**:
+
+```php
+$config[KernelConstants::CORE_NAMESPACES] = [ 'YourCompanyName', 'SprykerShop', 'SprykerEco', 'Spryker', 'SprykerSdk', ];
+```
+
+
+## Extend Spryker functionality from a module
+
+You can extend Spryker functionality from a module using one of the following options:
+
+* Introduce an extension point in an existing module. For an example, see [How to use a plugin from another module](docs/dg/dev/backend-development/plugins/plugins.html#how-to-use-a-plugin-from-another-module).
+* Use an existing extension point in a module. For instructions, see [Extend functionality using an existing extension point](#Extend-functionality using-an-existing-extension-point).
+
+### Extend functionality using an existing extension point
+
+1. Extend the product table with a **Categories** column that shows the list of categories, which a product is added to. For this, use the following extension points in the existing modules:
+  - [ProductTableConfigurationExpanderPluginInterface](https://github.com/spryker/product-management-extension/blob/master/src/Spryker/Zed/ProductManagementExtension/Dependency/Plugin/ProductTableConfigurationExpanderPluginInterface.php): to extend table headers with **Categories** column.
+  - [ProductTableDataBulkExpanderPluginInterface](https://github.com/spryker/product-management-extension/blob/master/src/Spryker/Zed/ProductManagementExtension/Dependency/Plugin/ProductTableDataBulkExpanderPluginInterface.php): to extend table content with the corresponding column
```

Comment:
Is this general information or what do I need to do with this?

Reply:
As far as I can see it was fixed by you already